README.md

```diff
@@ -17,3 +17,9 @@ $ make run-standalone
 ```
 
 To run using Krun, see the included Krun config file, `experiment.krun`.
+
+When running with Krun, the experiment directory will need to be writable by
+the krun user, as many benchmarks write stuff to disk. Ideally, we'd just grant
```

Comment:
I'm not sure I understand the "ideally" sentence, particularly the gnomic "that's not strictly correct" comment.

Reply:
If the krun user was permanent, then I'd just `chgrp krun` and `chown 775` the experiment dir, but there's no guarantee the next incarnation of the krun user will get the same GID. Granted, it mostly is the case, but it still makes me nervous.

Reply:
Ah, so the problem here is that we can't guarantee consistent UIDs and GIDs? That makes sense -- and I hadn't thought of that! Let's say that explicitly (but briefly).

Reply:
OK.